docs/advertising.md

```diff
@@ -0,0 +1,149 @@
+# Ad Layout and Waterfalling
+
+Ad layout is a new config option lets the application to sets a scheme of vasts (instead of VMAP) with waterfalling mechanism.
```

Comment:
In general, writing in docs about something being new will soon become obsolete, this is more appropriate to blog post that has a timestamp.

I would change the title to say 

> Advertisement layout management

And then the description body will say:

> With ad layout config you can create your own ad break timeline using your vast tags.
> Ad break can be set as pre, mid and post rolls and each ad break can contain a single vast tag or multiple tags, either as a pod, but also as a Waterfall*

And then add a short note saying "see what is waterfall in waterfall section" and add a short explainer there(maybe a task for Ron L)